crates/ra_hir/src/ty/.tests.rs.pending-snap

```diff
@@ -0,0 +1,28 @@
1) '3': isize\n[122; 123) 'd': [isize;usize]\n[126; 138) '[1, y, 2, 3]': [isize;usize]\n[127; 128) '1': isize\n[127; 128) '1': usize\n[130; 131) 'y': isize\n[133; 134) '2': isize\n[136; 137) '3': isize\n[148; 149) 'e': [isize;usize]\n[152; 155) '[y]': [isize;usize]\n[153; 154) 'y': isize\n[153; 154) 'y': isize\n[165; 166) 'f': [[isize;usize];usize]\n[169; 175) '[d, d]': [[isize;usize];usize]\n[170; 171) 'd': [isize;usize]\n[170; 171) 'd': [isize;usize]\n[173; 174) 'd': [isize;usize]\n[185; 186) 'g': [[isize;usize];usize]\n[189; 195) '[e, e]': [[isize;usize];usize]\n[190; 191) 'e': [isize;usize]\n[190; 191) 'e': [isize;usize]\n[193; 194) 'e': [isize;usize]\n[206; 207) 'h': [i32;usize]\n[210; 216) '[1, 2]': [i32;usize]\n[211; 212) '1': i32\n[211; 212) '1': usize\n[214; 215) '2': i32\n[226; 227) 'i': [&str;usize]\n[230; 240) '[\"a\", \"b\"]': [&str;usize]\n[231; 234) '\"a\"': &str\n[231; 234) '\"a\"': &str\n[236; 239) '\"b\"': &str\n[251; 252) 'b': [[&str;usize];usize]\n[255; 265) '[a, [\"b\"]]': [[&str;usize];usize]\n[256; 257) 'a': [&str;usize]\n[256; 257) 'a': [&str;usize]\n[259; 264) '[\"b\"]': [&str;usize]\n[260; 263) '\"b\"': &str\n[260; 263) '\"b\"': &str\n[275; 276) 'x': [u8;usize]\n[288; 290) '[]': [u8;usize]\n[300; 301) 'z': &[u8;usize]\n[311; 321) '&[1, 2, 3]': &[u8;usize]\n[312; 321) '[1, 2, 3]': [u8;usize]\n[313; 314) '1': u8\n[313; 314) '1': usize\n[316; 317) '2': u8\n[319; 320) '3': u8"},"old":{"module_name":"tests","metadata":{"created":"2019-04-03T20:30:58.397743505Z"},"snapshot":"\n[9; 10) 'x': &str\n[18; 19) 'y': isize\n[28; 324) '{     ... 3]; }': ()\n[38; 39) 'a': [&str]\n[42; 45) '[x]': [&str]\n[43; 44) 'x': &str\n[55; 56) 'b': [[&str]]\n[59; 65) '[a, a]': [[&str]]\n[60; 61) 'a': [&str]\n[63; 64) 'a': [&str]\n[75; 76) 'c': [[[&str]]]\n[79; 85) '[b, b]': [[[&str]]]\n[80; 81) 'b': [[&str]]\n[83; 84) 'b': [[&str]]\n[96; 97) 'd': [isize]\n[100; 112) '[y, 1, 2, 3]': [isize]\n[101; 102) 'y': isize\n[104; 105) '1': isize\n[107; 108) '2': isize\n[110; 111) '3': isize\n[122; 123) 'd': [isize]\n[126; 138) '[1, y, 2, 3]': [isize]\n[127; 128) '1': isize\n[130; 131) 'y': isize\n[133; 134) '2': isize\n[136; 137) '3': isize\n[148; 149) 'e': [isize]\n[152; 155) '[y]': [isize]\n[153; 154) 'y': isize\n[165; 166) 'f': [[isize]]\n[169; 175) '[d, d]': [[isize]]\n[170; 171) 'd': [isize]\n[173; 174) 'd': [isize]\n[185; 186) 'g': [[isize]]\n[189; 195) '[e, e]': [[isize]]\n[190; 191) 'e': [isize]\n[193; 194) 'e': [isize]\n[206; 207) 'h': [i32]\n[210; 216) '[1, 2]': [i32]\n[211; 212) '1': i32\n[214; 215) '2': i32\n[226; 227) 'i': [&str]\n[230; 240) '[\"a\", \"b\"]': [&str]\n[231; 234) '\"a\"': &str\n[236; 239) '\"b\"': &str\n[251; 252) 'b': [[&str]]\n[255; 265) '[a, [\"b\"]]': [[&str]]\n[256; 257) 'a': [&str]\n[259; 264) '[\"b\"]': [&str]\n[260; 263) '\"b\"': &str\n[275; 276) 'x': [u8]\n[288; 290) '[]': [u8]\n[300; 301) 'z': &[u8]\n[311; 321) '&[1, 2, 3]': &[u8]\n[312; 321) '[1, 2, 3]': [u8]\n[313; 314) '1': u8\n[316; 317) '2': u8\n[319; 320) '3': u8"}}
```

Comment:
this file should not be commited. I guess we should add .pending-snaps to gitignore?